.eslintrc.json

```diff
@@ -11,6 +11,8 @@
     "fetchMock": false
   },
   "rules": {
+    "jsdoc/no-undefined-types": "off",
+    "jsdoc/require-param": "off",
```

Comment:
These two rules shouldn't be turned off, instead the violations should be fixed by importing the relevant types / adding the necessary param docs.

Reply:
Ok, updated.